### PR TITLE
fix sonar issues in openshift.ui

### PR DESCRIPTION
**Pull request by adietish**
Signed-off-by: Andre Dietisheim <adietish@redhat.com>

# Pull Request Checklist
## General
* Is this a blocking issue or new feature? If yes, QE needs to +1 this PR

## Code
* Are method-/class-/variable-names meaningful?
* Are methods concise, not too long?
* Are catch blocks catching precise Exceptions only (no catch all)?

## Testing
* Are there unit-tests?
* Are there integration tests (or at least a jira to tackle these)?
* Is the non-happy path working, too?
* Are other parts that use the same component still working fine?

## Function
* Does it work?

### CODE REVIEW

**Comment by adietish** [comment]
@odockal dare to please review this? There's 0 functional change, only minimal changes to make sonar happy

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/common/AbstractProjectPage.java [hunk 1]
```diff
@@ -115,15 +115,12 @@ private void createProjectControls(Composite parent, DataBindingContext dbc) {
 		IObservableValue selectedProjectObservable = ViewerProperties.singleSelection().observe(projectsViewer);
 		Binding selectedProjectBinding = ValueBindingBuilder.bind(selectedProjectObservable)
 				.converting(new ObservableTreeItem2ModelConverter(IProject.class))
-				.validatingAfterConvert(new IValidator() {
-
-					@Override
-					public IStatus validate(Object value) {
+				.validatingAfterConvert((Object value) -> {
```
Comment (jkopriva) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/common/AbstractProjectPage.java]: can you please also remove import IValidator - Sonar reports it unused

**Comment by adietish** [comment]
@jkopriva addressed your finding and added further fixes that sonar kept piling up. Please re-review. Ideally we'd also run this against ITs.

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/BuildConfigTreeItems.java [hunk 2]
```diff
@@ -1,3 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Red Hat, Inc.
```
Comment (jkopriva) [inline plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/BuildConfigTreeItems.java]: I am not sure, if it is ok to have copyright year 2015 :(

**Comment by jkopriva** [comment]
@adietish ITests looks good, can you please also review sonar issues in your PR? there is another unused import and some other stuff. Otherwise we can merge.

File: plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/BuildConfigTreeItems.java (hunk 2 above)
Comment (adietish) [reply plugins/org.jboss.tools.openshift.ui/src/org/jboss/tools/openshift/internal/ui/wizard/importapp/BuildConfigTreeItems.java]: In commit history, it's 2015, but you're right. If we consider the current changes, 2019 is the more correct year :) Correcting.

**Comment by adietish** [comment]
@jkopriva thx for running the ITests! Addressed your complains and pushed it. Please re-review.

**Comment by jkopriva** [comment]
@adietish Sonar does not have any other complains, so merging. Thanks!

**Comment by adietish** [comment]
@jkopriva thx!

**Pull request merged**